site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -436,6 +437,14 @@ image::core-concepts/tenant-configuration-webhooks-settings.png[Tenant Configura
 
 include::docs/v1/tech/events-webhooks/_tenant-webhooks-table.adoc[]
 
+==== SCIM
+
```

Comment:
Yeah, was planning to add the screenshots etc. later. Just trying to get a first rev of SCIM stuff in instead of trying to do everything in one giant PR. It's already a lot.

Reply:
okay, sounds good. You can always add 

// TODO add screenshots 

and asciidoctor will omit the comment.

Reply:
Added // TODO